vtex-sites/storecomponents.store
# WIP | fix Sonarqube/Lint pipelines

## What's the purpose of this pull request?
Our Sonarqube pipeline is triggering some errors and warnings about Typescript ESLint issues, those changes will fix them.

## How to test it?
Those pipelines happens in our store projects. So, we can check if the pipeline will fail in this PR or [Marin Brasil PR](https://github.com/vtex-sites/marinbrasil.store/pull/289).

[Faststore Fix](https://github.com/vtex/faststore/pull/465)
